config.go

```diff
@@ -7,9 +7,89 @@ import (
 	"time"
 )
 
+// These are the versions of the protocol (which includes RPC messages as
+// well as Raft-specific log entries) that this server can _understand_. Use
+// the ProtocolVersion member of the Config object to control the version of
+// the protocol to use when _speaking_ to other servers. This is not currently
+// written into snapshots so they are unversioned. Note that depending on the
```

Comment:
Since the snapshots are somewhat divorced from Raft log entry types I figured it would be overkill. You can think of them as being at "snapshot format version 0" :-)